MaintenanceManager/MaintenanceManager.cpp

```diff
@@ -708,7 +711,11 @@ namespace WPEFramework {
             /* add 4 checks every 30 seconds */
             int i=0;
             do{
-                network_available = checkNetwork();
+#ifdef DISABLE_THUNDER_CLIENT_IN_MAINTENANCE_MANAGER_GTEST
```

Comment:
We should not make any changes to source code to support unit testing. If you want a null ptr, pls add a mock in unit test code. You can refer to other unit tests for reference.